README.md

```diff
@@ -6,9 +6,11 @@
 - <img src="./misc/icons/vue.png" width="16" alt="Vue icon"> [Live Stacks Vue Starter Template](https://stacks-vue.vercel.app)
 - <img src="./misc/icons/angular.png" width="16" alt="Angular icon"> [Live Stacks Angular Starter Template](https://stacks-angular.vercel.app)
 
-## Quick-Start a Stacks.js Project
+## Quick-Start a Stacks.js project
 
-The following command will spawn an interactive CLI, which allows you to select your favorite framework to use with Stacks.js and get started building in an instant! ⚡️
+This project aims to help quickly bootstrap frontend applications with Stacks.js on top of the desired JS framework as the foundation.
```

Comment:
Perhaps move this to the top and above the template bullets? So folks know right away what this project is about.